minecraft-server/start-minecraftFinalSetup

```diff
@@ -24,7 +24,7 @@ if [ -n "$ICON" -a ! -e server-icon.png ]; then
 fi
 
 # Make sure files exist and are valid JSON (for pre-1.12 to 1.12 upgrades)
-if [[ -z "ls *.json" ]]; then
+if [[ ! -z "find . -name '$*.json'" ]]; then
```

Comment:
seems like it should be

```
if [[ $(find . -name '*.json' | wc) != 0 ]]; then
```

since `find` seems to have a success result code whether it found files or not.